addons/binding/org.openhab.binding.modbus/hs_err_pid5264.log

```diff
@@ -0,0 +1,1915 @@
+#
```

Comment:
You can reduce your PR with 1915 lines of "code" (6%) by just removing this file. :-)

Reply:
Done